python/ingestion/gcs_to_bq_util.py

```diff
@@ -109,7 +110,10 @@ def load_csv_as_dataframe(gcs_bucket, filename, dtype=None):
   blob = bucket.blob(filename)
   local_path = local_file_path(filename)
   blob.download_to_filename(local_path)
-  return pandas.read_csv(local_path, dtype=dtype)
+  frame = pandas.read_csv(local_path, dtype=dtype)
+
+  os.remove(local_path)
```

Comment:
Does this file get cleaned up automatically?

Reply:
No, read_csv leaves the file as is and Cloud Run reuses instances and keeps the same local filesystem. You just can't guarantee which instance a request will go to if there are multiple.